test/common/helpers.test.js

```diff
@@ -88,6 +88,48 @@ test('should parse "drs://dg." Data Object uri with only a host part with a quer
  * End Scenario 3
  */
 
+/**
+ * Begin Scenario 4: data objects uri with jade data repo host
+ */
+test('should parse Data Object uri with jade data repo DEV as host', (t) => {
+    t.is(
+        dataObjectUriToHttps('drs://jade.datarepo-dev.broadinstitute.org/973b5e79-6433-40ce-bf38-686ab7f17820'),
+        'https://jade.datarepo-dev.broadinstitute.org/ga4gh/drs/v1/objects/973b5e79-6433-40ce-bf38-686ab7f17820'
+    );
+});
+
+test('should parse Data Object uri with jade data repo DEV as host and path with snapshot id', (t) => {
+    t.is(
+        dataObjectUriToHttps('drs://jade.datarepo-dev.broadinstitute.org/v1_c78919df-5d71-414b-ad29-7c3c0d810657_973b5e79-6433-40ce-bf38-686ab7f17820'),
+        'https://jade.datarepo-dev.broadinstitute.org/ga4gh/drs/v1/objects/v1_c78919df-5d71-414b-ad29-7c3c0d810657_973b5e79-6433-40ce-bf38-686ab7f17820'
+    );
+});
+
+test('should parse Data Object uri with jade data repo PROD as host', (t) => {
+    t.is(
+        dataObjectUriToHttps('drs://jade-terra.datarepo-prod.broadinstitute.org/anything'),
+        'https://jade-terra.datarepo-prod.broadinstitute.org/ga4gh/drs/v1/objects/anything'
+    );
+});
+
+test('should throw error when given jade data repo host and no path', (t) => {
+    try {
+        dataObjectUriToHttps('drs://jade.datarepo-dev.broadinstitute.org/');
+    } catch(error) {
```

Comment:
What happens in this test when an error is not thrown?

I don't know this test kit yet, but should there be something like `fail("Should have thrown an error but didn't")` before the `catch`?

Reply:
I am not sure either. I will look into it.

Reply:
There is `t.throws(fn, expectation?, message?)` which can be used. If the test fails to throw an error it will fail the test and return the message passed. I will update the code accordingly.